docs/reference/commandline/network_inspect.md

```diff
@@ -201,6 +201,97 @@ $ docker network inspect ingress
 ]
 ```
 
+### Using `verbose` option for `network inspect`
+
+`docker network inspect --verbose` for swarm mode overlay networks shows service specific details like VIP, Port mapping along 
+with IPs of tasks and the IP of the swarm node where the task is running.
```

Comment:
"...shows service-specific details like VIPs and port mappings. It also shows IPs of service tasks, and the IPs of the nodes where the tasks are running."